internal/controlplane/handlers_githubwebhooks_test.go

```diff
@@ -239,27 +236,6 @@ func (s *UnitTestSuite) TestHandleWebHookRepository() {
 	repositoryID := uuid.New()
 	projectID := uuid.New()
 
-	// create a test oauth2 token
```

Comment:
These are no longer needed. Previously, these operations get called to create a ProviderBuilder. In the TraitInstantiator, they are only called when a dependent attempts to instantiate a provider traits. Since these test cases do not need a GitHub instance to be instantiated, they are unused.